repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/series.ftl

```diff
@@ -19,18 +19,34 @@
                 <h2><@fmt.message key="headers.summary"/></h2>
                 <@structuredText item=series.summary uipath="ps.series.summary" />
 
-                <ul class="simple-list simple-list--publications" data-uipath="ps.series.publications-list"><#list publications as publication>
-                    <li>
-                        <a href="<@hst.link hippobean=publication.selfLinkBean/>"
-                            title="${publication.title}">
-                            ${publication.title}
-                        </a>
-                        <p><@truncate text=publication.summary.firstParagraph size="300"/></p>
-                    </li>
-                </#list></ul>
+                <#if publications?has_content>
+                    <h3 class="flush push--bottom"><@fmt.message key="headers.latest-version"/></h3>
+                    <ul class="simple-list simple-list--publications" data-uipath="ps.series.publications-list.latest">
+                        <@publicationItem publication=publications?first/>
+                    </ul>
+                </#if>
+
+                <#if publications?size gt 1>
+                    <h3 class="flush push--bottom"><@fmt.message key="headers.previous-versions"/></h3>
+                    <ul class="simple-list simple-list--publications" data-uipath="ps.series.publications-list.previous">
+                        <#list publications as publication>
```

Comment:
Hmm, I may be wrong but it appears that, apart of being displayed in the dedicated 'Latest Publication' section the latest publication will also be displayed at the top of the 'Previous Publications' section. Have I misread it?

Reply:
🙈  Good spot

Reply:
Fixed thanks to @wojtek-oledzki 😁